packages/model-viewer/src/three-components/gltf-instance/gltf-2.0.ts

```diff
@@ -78,23 +80,38 @@ export interface OrthographicCamera {
 
 export type Camera = PerspectiveCamera|OrthographicCamera;
 
-export type NearestFilter = 9728;
-export type LinearFilter = 9729;
-export type NearestMipmapNearestFilter = 9984;
-export type LinearMipmapNearestFilter = 9985;
-export type NearestMipmapLinearFilter = 9986;
-export type LinearMipmapLinearFilter = 9987;
+export enum Filter {
+  Nearest = 9728,
+  Linear = 9729,
+  NearestMipmapNearest = 9984,
+  LinearMipmapNearest = 9985,
+  NearestMipmapLinear = 9986,
+  LinearMipmapLinear = 9987,
+}
+
+export type NearestFilter = Filter.Nearest;
+export type LinearFilter = Filter.Linear;
+export type NearestMipmapNearestFilter = Filter.NearestMipmapNearest;
+export type LinearMipmapNearestFilter = Filter.LinearMipmapNearest;
+export type NearestMipmapLinearFilter = Filter.NearestMipmapLinear;
+export type LinearMipmapLinearFilter = Filter.LinearMipmapLinear;
```

Comment:
Do we really need both an enum and the individual constants?

Reply:
I removed them and it created several type conversion issues elsewhere, it started to creep out of scope so I backed off. But happy to lay waste to that area of code perhaps in a different CL?